### DISCUSSION TITLE
Help optimizing a vercel app

### DISCUSSION THREAD

**Discussion by counterspace27** [vercel/community#7153] [Help]
### Summary

Hello,
We are looking for a vercel expert to assist us with optimizing our app/project on vercel. The goal is to implement cache, and optimize the site so that it be the best it can be on vercel

### Steps to Reproduce

Nothing to reproduce, the effort will be paid to assist with optimizations!

**Comment by versecafe** [comment]
@counterspace27 What is your project built upon and are you looking for simple request caching, or a more comprehensive request + resource + page pregeneration + source side caching & DB indexing setup?

**Reply by counterspace27** [reply]
It's a wordpress headless integration using frontity for the vercel app. I believe we are looking for the later and more comprehensive.

**Reply by counterspace27** [reply]
This would be a budget project…

**Reply by versecafe** [reply]
If you want I can set up a meeting to go over everything with you, also looking into frontity you may want to try migrating to something like NextJS and rolling a integration with headless Wordpress since frontity is not actively maintained anymore, and getting access to unstable_cache with Next can allow some very effective granular caching over DB reads and data mutation, plus the general improvements of RSCs, Nexts caching layer, Prerendering, etc.

**Reply by counterspace27** [reply]
A meeting sounds great, and good to know about frontity. My team was already exploring other options, so it will be good to hear a different perspective. We are EST timezone, and have the afternoons after 1pm available.

**Reply by versecafe** [reply]
@counterspace27 Are you free for 5pm EST?

**Reply by counterspace27** [reply]
@versecafe Apologies I was at a meeting. But tomorrow after 1130am I am free until 4pm

**Reply by versecafe** [reply]
Can you give me an email to reach out to or just leave something like a google meet link but 2pm tomorrow would work great.

**Reply by counterspace27** [reply]
Of course, brett.collins@counterspace.us and we can connect from there

[a comment by an automated account is not shown]
